packages/lib/package.json

```diff
@@ -45,7 +45,8 @@
         "type-check-generate": "tsc --emitDeclarationOnly",
         "lint": "eslint 'src/**/*.{js,ts,tsx}' --quiet",
         "lint:fix": "npm run lint -- --fix",
-        "prepublishOnly": "npm run build"
+        "prepublishOnly": "npm run build",
+        "prepare": "cd ../.. && husky install packages/lib/.husky"
```

Comment:
How does it work to setup the commit hooks in our machines with VM? Do we need to run this script/do any other setup or will it work out of the box?

Reply:
Yeah, it does. There should be no difference since you are running already inside of vagrant. Just the commits are outside.